scss/index.scss

```diff
@@ -48,3 +48,4 @@ p {
 @import './challengeMaterial.scss';
 @import './modalCard.scss';
 @import './prism.scss';
+@import './modalImage.module.scss';
```

Comment:
Unfortunately this needs to be global because it modifies 3rd party components (modal Image uses `react-bootsrap` component`).